packages/flutter_test/lib/src/test_async_utils.dart

```diff
@@ -73,34 +73,38 @@ class TestAsyncUtils {
       assert(_scopeStack.contains(scope));
       bool leaked = false;
       _AsyncScope closedScope;
-      final StringBuffer message = StringBuffer();
+      final List<DiagnosticsNode> information = <DiagnosticsNode>[];
       while (_scopeStack.isNotEmpty) {
         closedScope = _scopeStack.removeLast();
         if (closedScope == scope)
           break;
+        // TODO(jacobr): what is the ErrorSummary for this error?
```

Comment:
it's what you have as the description below, "Asynchronous call to guarded function leaked."

Reply:
Agreed. That is the summary. Done.